README.adoc

```diff
@@ -1,5 +1,16 @@
 = C++ example project scanned on SonarQube using GitHub Actions
 
+// URIs:
+:uri-qg-status: https://next.sonarqube.com/sonarqube/dashboard?id=sonarsource-cfamily-examples_linux-autotools-gh-actions-sq_AYAYt8D0y0k_ZlpkA-ln
+:img-qg-status: https://next.sonarqube.com/sonarqube/api/project_badges/measure?project=sonarsource-cfamily-examples_linux-autotools-gh-actions-sq_AYAYt8D0y0k_ZlpkA-ln&metric=alert_status&token=squ_a0683d6d23bc3fa8b93a6befc840c774511333cc
+:uri-build-status: https://github.com/sonarsource-cfamily-examples/linux-autotools-gh-actions-sq/actions/workflows/main.yml
+:img-build-status: https://github.com/sonarsource-cfamily-examples/linux-autotools-gh-actions-sq/actions/workflows/main.yml/badge.svg
+
+image:{img-build-status}[Build Status, link={uri-build-status}]
+image:{img-qg-status}[Quality Gate Status,link={uri-qg-status}]
+
+*This project is analysed on https://next.sonarqube.com/sonarqube/dashboard?id=sonarsource-cfamily-examples_linux-autotools-gh-actions-sq_AYAYt8D0y0k_ZlpkA-ln[SonarQube]!*
```

Comment:
Doc links got outdated anchors: `header-4`, `header-7`, `header-8` got shifted, because, I suppose a new section was added to SQ documentation. I think it makes sense to update them as part of this "onboarding" series of PRs

Reply:
Good catch!
Makes sense, I'll update them